src/st/no3/377D4.c

```diff
@@ -463,19 +488,31 @@ s16 func_801C5560(s32 arg0) { return D_801820C4[arg0 & 0xFF]; }
 
 INCLUDE_ASM("asm/st/no3/nonmatchings/377D4", func_801C557C);
 
-INCLUDE_ASM("asm/st/no3/nonmatchings/377D4", func_801C55E8);
+s32 func_801C55E8(s16 arg0, s16 arg1) {
```

Comment:
This function probably actually returns u8, and the & 0xFF at the end can be gotten rid of